### PR TITLE
add hourly forecast

### PR DESCRIPTION
**Pull request by Obleo33**
next 7 hours will now append to the dom

### CODE REVIEW

File: lib/components/Hourly.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+const Hourly = ({ weatherHourly }) => {
+  if(weatherHourly.length<1) {
+    return(
+      <div></div>
+    )
+  } else {
+    return(
+      <div className='hourly-forcast'>
+      {weatherHourly.map((a,i) => {
```
Comment (Obleo33) [inline lib/components/Hourly.js:11]: Maps through the array and appends to the dom

File: lib/components/Hourly.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const Hourly = ({ weatherHourly }) => {
+  if(weatherHourly.length<1) {
```
Comment (Obleo33) [inline lib/components/Hourly.js:4]: if weatherHourly is empty append an empty div

File: lib/components/Main.js [hunk 3]
```diff
@@ -5,11 +5,11 @@ class Main extends Component {
   render() {
     return (
       <div>
-        <Weathrly source='http://api.wunderground.com/api/395c3eaa19684a65/forecast/q/' />
+        <Weathrly source='http://api.wunderground.com/api/395c3eaa19684a65/' />
```
Comment (Obleo33) [inline lib/components/Main.js:8]: adjusted the source so we can dip into the hourly and 10 day

**Comment by hmorri32** [comment]
Sweet.

**Pull request merged**